feat(blog): Theme Jam wrap-up post (#16440)
gatsbyjs/gatsby: docs/blog/2019-08-07-theme-jam/index.md

---
title: "Gatsby Theme Jam: Congratulations Vojtěch Holík and Allan Pooley!"
date: 2019-08-07T12:00:00
author: Jason Lengstorf
excerpt: >

>   During the Theme Jam, 111 themes wer created by our incredible community. Join us in celebrating the Gatsby community (that’s you!) and congratulating Vojtěch and Allan!

Reviewer: ```suggestion
  During the Theme Jam, 111 themes were created by our incredible community. Join us in celebrating the Gatsby community (that’s you!) and congratulating Vojtěch and Allan!
```

tags:
  - themes
  - community
---

At the beginning of July, [Brent Jackson](https://twitter.com/jxnblk) joined _Learn With Jason_ to show [how to use `theme-ui`](https://youtu.be/6Z4p-qjnKCQ?list=PLz8Iz-Fnk_eTpvd49Sa77NiF8Uqq5Iykx). During the show, we announced the stable launch of [Gatsby themes](/docs/themes/) and kicked off the [Theme Jam](https://themejam.gatsbyjs.org), a celebration of our incredible community of creators.

## What is the Theme Jam?

> In a nutshell, the Theme Jam was a way for Gatsby to give back to its community: everyone who published a theme and submitted it to the Theme Jam will receive exclusive swag designed by [Maggie Appleton](https://maggieappleton.com/), and two creators who published themes our team reviewed as exemplary are getting an all-expenses paid trip to [Gatsby Days](https://www.gatsbyjs.com/resources/gatsby-days/). (For more details, see the [rules](https://themejam.gatsbyjs.org/rules).)

Reviewer: Maybe we could split this into two sentences?

Author: Good call. Done!

tags:
  - themes
  - community
---

At the beginning of July, [Brent Jackson](https://twitter.com/jxnblk) joined _Learn With Jason_ to show [how to use `theme-ui`](https://youtu.be/6Z4p-qjnKCQ?list=PLz8Iz-Fnk_eTpvd49Sa77NiF8Uqq5Iykx). During the show, we announced the stable launch of [Gatsby themes](/docs/themes/) and kicked off the [Theme Jam](https://themejam.gatsbyjs.org), a celebration of our incredible community of creators.

## What is the Theme Jam?

![Theme Jam illustration by Maggie Appleton.](images/theme-jam.png)

In a nutshell, the Theme Jam was a way for Gatsby to give back to its community: everyone who published a theme and submitted it to the Theme Jam will receive exclusive swag designed by [Maggie Appleton](https://maggieappleton.com/). Two creators who published themes our team reviewed as exemplary are getting an all-expenses paid trip to [Gatsby Days](https://www.gatsbyjs.com/resources/gatsby-days/). (For more details, see the [rules](https://themejam.gatsbyjs.org/rules).)

## How were entries reviewed?

Every Theme Jam entry needed to have a demo site, a published npm package, a public repo, and a properly configured `package.json`.

> Entries also needed to be incredibly performant (something something blazing fast) and meet accessibility standards.

Reviewer 2: ```suggestion
Entries also needed to be incredibly performant (_something something blazing fast_) and meet accessibility standards.
```

On top of that, submissions were code reviewed for documentation, the ability to extend and shadow the themes, and creativity.

> ### Part 1: automated review

Reviewer 2: ```suggestion
### Part 1: Automated Review
```

Is there a reason these are lower-cased?

Author: Force of habit. I didn't check the style guide. My own style is to title case H2s, but sentence case H3 and below.

Several parts of the judging could be done automatically, so we set up an ugly-but-effective [automated test](https://github.com/jlengstorf/theme-jam-tester) that would:

1. Run a [webpagetest.org](https://webpagetest.org/) audit on the demo site
2. Run a [Lighthouse](https://developers.google.com/web/tools/lighthouse/) audit on the demo site
3. Check for required fields and keywords in `package.json` (shout out to [unpkg.com](https://unpkg.com) for making this easy)

To pass the automated review, a theme must:

- score at least 90 on the Lighthouse accessibility audit
- score at least 80 on the Lighthouse performance audit
- have all required fields in `package.json`

Of the 111 submissions, 52 passed the automated review.

> ### Part 2: manual review

Reviewer 2: ```suggestion
### Part 2: Manual Review
```

> Every theme that cleared the automated review was manually reviewed by [Jackson](https://twitter.com/jxnblk), [Amberley](https://twitter.com/amber1ey), [John](https://twitter.com/4lpine), [Kyle Gill](https://gill_kyle), [Chris](https://twitter.com/chrisbiscardi), [Flo](https://twitter.com/fk), and me.

Reviewer 2: ```suggestion
Every theme that cleared the automated review was manually reviewed by [Jackson](https://twitter.com/jxnblk), [Amberley](https://twitter.com/amber1ey), [John](https://twitter.com/4lpine), [Kyle Gill](https://gill_kyle), [Chris](https://twitter.com/chrisbiscardi), [Flo](https://twitter.com/fk), and [me](https://twitter.com/jlengstorf).
```

> Amberley did a manual accessibility audit. All of the themes we reviewed has a few accessibility issues, so we will be opening issues with feedback and suggestions to improve that.

Reviewer 2: ```suggestion
Amberley did a manual accessibility audit. Nearly all of the themes we reviewed have a few accessibility issues, so we will be opening issues with feedback and suggestions to improve that.
```

We also looked for coding best practices, ease of [shadowing](/blog/2019-04-29-component-shadowing/)/extending the theme, documentation in the README, and overall creativity/ambitiousness of the project.

## Highlighting Exemplary Creators

We plan to feature a lot of the excellent creators and their themes that were created both as part of Theme Jam and outside of it.

For Theme Jam, we specifically wanted to call out excellence. This is really important with themes being so early; great examples lead to improving the quality of the entire ecosystem.

Two of the themes we received stood out especially. As we reviewed, we all took notes on the themes we thought were outstanding, and _every one of us_ chose these two themes as excellent examples.

> **So please join us in congratulating Vojtěch Holík and Allan Pooley, who will receive an all-expenses paid trip to Gatsby Days in recognition of their outstanding work on [`gatsby-theme-simplecast`](https://github.com/vojtaholik/gatsby-theme-simplecast) and [`gatsby-theme-legals-prismic`](https://github.com/littleplusbig/gatsby-theme-legals-prismic)!**

Reviewer 2: ```suggestion
**So please join us in congratulating Vojtěch Holík and Allan Pooley on their outstanding work on [`gatsby-theme-simplecast`](https://github.com/vojtaholik/gatsby-theme-simplecast) and [`gatsby-theme-legals-prismic`](https://github.com/littleplusbig/gatsby-theme-legals-prismic)!**
```

Reviewer 2: I (personally) don't really see a need to highlight Gatsby Days again, the focus is on their work.
